hrui/js/src/hrui/main.js

```diff
@@ -65,17 +65,15 @@
     /**
      * Amends the contact page and the contact form
      */
-    function amendContactPageAndForm (e) {
-      if (CRM.formName === 'contactForm' || CRM.pageName === 'viewSummary') {
-        // Rename "Summary" tab to "Personal Details"
-        // Hack to check contact type - This field only appears for individuals
-        if ($('.crm-contact-job_title', '.crm-summary-contactinfo-block').length) {
-          $('.crm-contact-tabs-list #tab_summary a', e.target).text('Personal Details');
```

Comment:
wasn't `e.target` important to limit the scope of this selector?

Reply:
I have mentioned the same in the PR doc
> Parameters has been removed from the following functions as that is not necessary, and code works without that.